README.md

```diff
@@ -31,6 +31,18 @@
   eval (fnm env --fish)
   ```
 
+### install reference
+
+```bash
+cd ~
+wget https://github.com/Schniz/fnm/releases/download/v1.1.0/fnm-linux.zip
```

Comment:
Maybe we should always download the latest version? here's a gist that helps with that: https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c#file-get_latest_release-sh-L2-L6

```bash
get_latest_release() {
  curl --silent "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
    grep '"tag_name":' |                                            # Get tag line
    sed -E 's/.*"([^"]+)".*/\1/'                                    # Pluck JSON value
}

LATEST_RELEASE=$(get_latest_release Schniz/fnm)
URL=https://github.com/Schniz/fnm/releases/download/$LATEST_RELEASE/fnm-linux.zip
```

Also, do you think we should have one script for both Mac and Linux?